lib/ansible/module_utils/eos.py

```diff
@@ -50,6 +50,7 @@
     'auth_pass': dict(no_log=True, fallback=(env_fallback, ['ANSIBLE_NET_AUTH_PASS'])),
 
     'use_ssl': dict(default=True, type='bool'),
+    'use_proxy': dict(default=True, type='bool'),
```

Comment:
Can you please add documentation for `use_proxy` [here](https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/module_docs_fragments/eos.py)